docs/cristin.ttl

```diff
@@ -503,7 +546,7 @@
 
 :isbn
     a               :cristinfield ;
-    rdfs:comment    "The isbn which the respective book is registered with." ;
+    rdfs:comment    "The isbn which the respective book is registered with. Could be nullable if the class also is a tidsskriftserie" ;
```

Comment:
Is nullable when the domain is :TIDSSKRIFTSERIE_Class. We should be more consistent with the way we write classes, otherwise it will be very difficult to spot all references of a a class